datacapturing/src/main/java/de/cyface/datacapturing/backend/DataCapturingBackgroundService.java

```diff
@@ -150,11 +151,33 @@
      * The last captured {@link GeoLocation} used to calculate the distance to the next {@code GeoLocation}.
      */
     private GeoLocation lastLocation = null;
+    /**
+     * The {@link Measurement#distance} in meters until the last location update.
+     */
     private double lastDistance;
+    /**
+     * A device-wide unique identifier for the application containing this SDK such as
+     * {@code Context#getPackageName()} which is required to generate unique global broadcasts for this app.
+     * <p>
+     * <b>Attention:</b> The identifier must be identical in the global broadcast sender and receiver.
+     */
+    private String appId;
 
     @Override
     public IBinder onBind(final @NonNull Intent intent) {
         Log.v(TAG, String.format("Binding to %s", this.getClass().getName()));
+
+        // FIXME [MOV-660]: Is onBind() called more often than onStartCommand()? If so, we need to make sure Ping is
```

Comment:
Hier bin ich mir gerade (langer Tag, spät Abends) nicht mehr sicher:

**Wird vom BGS erwartet, dass er einen PingReceiver jedes mal neu stellt bei jedem bind (siehe unten) und diesen registriert?
Können wir hierdurch mehrere PingReceiver haben was schlecht wäre?**

bindService() is called in:
1. DCS.bind() and this is called by:
1.1 DCS.reconnect()
1.2 DCS.runService() - pingReceiver expected
2. DCS.onBindingDied() to rebind